doc/user_doc/vic_installation/introduction.md

```diff
@@ -1,50 +1,154 @@
 
-### vSphere Integrated Containers Engine VCH
+The format of this packaging, is called an *Image.* An image is a template from which many containers can be instantiated. The Docker image format allows for images to be composed in a parent child-relationship, just like a disk snapshot. This image hierarchy allows for common dependencies to be shared between containers. For example, I may have a Debian 8 image which has a child image with Java installed. That Java image may itself have a child with Tomcat installed. The Debian 8 image may have other children, such as PHP, Python etc. 
 
-vSphere Integrated Containers Engine containers run as virtual machines. The VCH is not a VM, but a vApp, which is a kind of resource pool. It is an abstract dynamic resource boundary defined and controlled by vSphere into which you can provision container VMs. The VCH can be a subset of a physical host or a subset of a cluster of hosts.
+The immutability of the image format means that you never modify an image, you always create a new one. The layered nature of the image format means that you can cache commonly-used layers and only have to download/upload layers you don’t already have. It also means that if you want to patch a particular image, you create a new one and then rebuild all of its children. 
 
-A one to one coupling exists between a container and a virtual machine. A container image is attached to the VM as a disk, the VM is either booted or forked from the kernel ISO, then the containerVM chroots into the container filesystem, effectively becoming the container.
+The beauty of the image format is its portability. So long as you have a destination running a container engine (such as Docker) and a compatible kernel ABI, then you can download and run an image on it. This portability is facilitated by a *Registry* which is a service that indexes and stores images. You can run your own private image registry that forms part of a development pipeline: Images are *pushed* to the registry from development, *pulled* to a test environment, verified and can then be *pulled* to a production environment.
 
-## VCH Deployment and Management ##
+**What is VIC**
 
-vSphere Integrated Containers Engine provides a command-line utility, `vic-machine`, that you use to deploy and manage VCHs. The different commands of the vic-machine utility allow you to perform the following actions:
+The VIC product portfolio includes a container engine (VICe), management portal (Admiral) and container registry (Harbor) that all currently support the Docker image format. VIC is entirely Open Source, free to use and support is included in the vSphere Enterprise Plus license.
```

Comment:
I disagree. If the poor old vSphere Admin is coming to VIC with no experience of containers, then the definitions of the VIC components make little sense if you haven't had the intro to containers first....